chapters/publication.tex

```diff
@@ -84,41 +81,39 @@ \subsection{Preparing dynamic documents}
 that a mistake will be made or something will be missed.
 Therefore this is a broadly unsuitable way to prepare technical documents.
 
-There are a number of tools that can be used for dynamic documents.
-Some are code-based tools such as R's RMarkdown\sidenote{\url{https://rmarkdown.rstudio.com}}
+However, the most widely utilized software
+for dynamically managing both text and results is \LaTeX\ (pronounced ``lah-tek'').\sidenote{
+	\url{https://github.com/worldbank/DIME-LaTeX-Templates}}
+\index{\LaTeX}
+\LaTeX\ is a document preparation and typesetting system with a unique syntax.
+While this tool has a significant learning curve,
+its enormous flexibility in terms of operation, collaboration, output formatting, and styling
+make it the primary choice for most large technical outputs.
+In fact, \LaTeX\ operates behind-the-scenes in many other dynamic document tools (discussed below).
+Therefore, we recommend that you learn to use \LaTeX\ directly
+as soon as you are able to and provide several resources for doing so in the next section.
+
+There are also code-based tools that can be used for dynamic documents, 
+such as R's RMarkdown\sidenote{\url{https://rmarkdown.rstudio.com}}
 and Stata's \texttt{dyndoc}.\sidenote{\url{https://www.stata.com/manuals/rptdyndoc.pdf}}
 These tools ``knit'' or ``weave'' text and code together,
 and are programmed to insert code outputs in pre-specified locations.
 Documents called ``notebooks'' (such as Jupyter\sidenote{\url{https://jupyter.org}}) work similarly,
 as they also use the underlying analytical software to create the document.
-These types of dynamic documents are usually appropriate for short or informal materials
+These tools are usually appropriate for short or informal documents
 because they tend to offer restricted editability outside the base software
```

Comment:
I am not fond of "base software". 

```suggestion
because it tends to be difficult to edit the content unless using the tool 
```
If this edits is accepted then next part of this sentence needs to be updated